tests/test_metadata.py

```diff
@@ -53,8 +51,4 @@ def test_docinfo(app, status, warning):
         'orphan': u'',
         'nocomments': u'',
     }
-    # I like this way of comparing dicts - easier to see the error.
-    for key in exampledocinfo:
-        yield assert_equal, exampledocinfo.get(key), expecteddocinfo.get(key)
-    # but then we still have to check for missing keys
-    yield assert_equal, set(expecteddocinfo.keys()), set(exampledocinfo.keys())
+    assert exampledocinfo == expecteddocinfo
```

Comment:
When test was be failure, pytest will show the error as:
```
===================================================================================================================================================== FAILURES =====================================================================================================================================================
___________________________________________________________________________________________________________________________________________________ test_docinfo ___________________________________________________________________________________________________________________________________________________

app = <SphinxTestApp buildername='pseudoxml'>, status = <_io.StringIO object at 0x7f90a0ea2d38>, warning = <_io.StringIO object at 0x7f90a0e45f78>

    @with_app('pseudoxml')
    def test_docinfo(app, status, warning):
        """
        Inspect the 'docinfo' metadata stored in the first node of the document.
        Note this doesn't give us access to data stored in subsequence blocks
        that might be considered document metadata, such as 'abstract' or
        'dedication' blocks, or the 'meta' role. Doing otherwise is probably more
        messing with the internals of sphinx than this rare use case merits.
        """
        app.builder.build(['metadata'])
        env = app.env
        exampledocinfo = env.metadata['metadata']
        expecteddocinfo = {
            'author': u'David Goodger',
            'authors': [u'Me', u'MYself', u'I'],
            'address': u'123 Example Street\nExample, EX  Canada\nA1B 2C3',
            'field name': u'This is a generic bibliographic field.',
            'field name 2': (u'Generic bibliographic fields may contain multiple '
                             u'body elements.\n\nLike this.'),
            'status': u'This is a "work in progress"',
            'version': u'1',
            'copyright': (u'This document has been placed in the public domain. '
                          u'You\nmay do with it as you wish. You may copy, modify,'
                          u'\nredistribute, reattribute, sell, buy, rent, lease,\n'
                          u'destroy, or improve it, quote it at length, excerpt,\n'
                          u'incorporate, collate, fold, staple, or mutilate it, or '
                          u'do\nanything else to it that your or anyone else\'s '
                          u'heart\ndesires.'),
            'contact': u'goodger@python.org',
            'date': u'2006-05-21',
            'organization': u'humankind',
            'revision': u'4564',
            'tocdepth': 1,
            'orphan': u'',
            'nocomments': u'',
        }
>       assert exampledocinfo == expecteddocinfo
E       assert {'address': '...hon.org', ...} == {'address': '1...hon.org', ...}
E         Omitting 14 identical items, use -v to show
E         Differing items:
E         {'authors': ['Me', 'Myself', 'I']} != {'authors': ['Me', 'MYself', 'I']}
E         Use -v to get the full diff

tests/test_metadata.py:54: AssertionError
============================================================================================================================================= 1 failed in 3.49 seconds =============================================================================================================================================
```